src/core/user/index.ts

```diff
@@ -87,32 +87,46 @@ export class Cookie extends Store {
     } as Required<CookieOptions>
   }
 
-  private opts(): jar.CookieAttributes {
+  private opts(): cookies.CookieAttributes {
     return {
-      sameSite: this.options.sameSite as jar.CookieAttributes['sameSite'],
+      sameSite: this.options.sameSite as cookies.CookieAttributes['sameSite'],
       expires: this.options.maxage,
       domain: this.options.domain,
       path: this.options.path,
     }
   }
 
   get<T>(key: string): T | null {
-    return jar.getJSON(key)
+    try {
+      const value = cookies.get(key)
+
+      if (!value) {
+        return null
+      }
+
+      try {
+        return JSON.parse(value)
+      } catch (e) {
+        return (value as unknown) as T
+      }
```

Comment:
Just curious - how do we handle JSON we can't parse later on, or is it not a big deal?

Reply:
I assumed that JSON that can't be parsed is "not" JSON 😄 

I kinda stole the logic from the previous implementation in js-cookie ([here](https://github.com/js-cookie/js-cookie/blob/54962f884e9ae33f93e13ac903ffaf1d5a523598/src/js.cookie.js#L125))